example/spec/parallel/syncGranule/SyncGranuleSuccessSpec.js

```diff
@@ -1,5 +1,5 @@
 const fs = require('fs');
-const difference = require('lodash.difference');
+const { difference } = require('lodash/difference');
```

Comment:
Shouldn't this be ...?
```suggestion
const difference = require('lodash/difference');
```

Reply:
The export is `export default difference`